data/100.yaml

```diff
@@ -170,11 +170,11 @@
       - author :
    51:
       - title  : Wilson’s Theorem
-      - decl   : wilsons_lemma
+      - decl   : number_theory/quadratic_reciprocity.html#zmod.wilsons_lemma
       - author : Chris Hughes
    52:
       - title  : The Number of Subsets of a Set
-      - decl   : card_powerset
+      - decl   : data/multiset.html#multiset.card_powerset
```

Comment:
I'd list `finset` version here.

Reply:
Seems correct. Once I figure out how to update it, it should be
data/finset.html#finset.card_powerset

Reply:
Does this do it
```suggestion
      - decl   : data/finset.html#finset.card_powerset
```